src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
Oh wow, that's so strange. I know this is kind of annoying, but you could submit the `#[path]` changes in a separate PR (including the separation of `helpers.rs`) so that I have a way of referring back to that particular change? I'd then be happy to merge both that and this.